client/src/components/FullLogo/Logo.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "./Logo.css";
+
+const Logo = props => {
+  let classList = "logo-tag";
+
+  let types = ["logo", "notag"];
+
+  if (types.includes(props.type)) {
+    classList += ` ${props.type}`;
+  }
+
+  if (props.notag) {
+    classList += ` notag` // Note the spacing here since we are adding to the string!
+}
+
+  return (
+    <>
+      <div className="logo-tag">
+        <div className="logo">
+          <div className="modern">Modern</div>
+          <div className="trousseau">trousseau</div>
```

Comment:
You should not put text inside a div tag. Text should be housed inside a `p` or `h` tag. If you need the block scoping of a `<div>` tag then you should wrap the `p` or `h` tag in a `div`. 

Example: 
```jsx
<div>
   <p className="modern">Modern</p>
</div>
```

Reply:
Modified the tags